core/src/main/java/org/infinispan/interceptors/impl/DistCacheWriterInterceptor.java

```diff
@@ -88,7 +91,9 @@ public Object visitPutMapCommand(InvocationContext ctx, PutMapCommand command) t
          PutMapCommand putMapCommand = (PutMapCommand) rCommand;
          LocalizedCacheTopology cacheTopology = dm.getCacheTopology();
          Map<Object, Object> map = putMapCommand.getMap();
-         int count = 0;
+
```

Comment:
@ryanemerson I'm not sure how this looked before @rvansa's comment, but please don't return a lambda from this method: just put the lambda body here, and then `protected final InvocationSuccessAction handlePutMapCommandReturn = this::handlePutMapCommandReturn`.